### PR TITLE
Add upgrade note for Symfony+Translatable setup

### PR DESCRIPTION
**Pull request by much-rebel**
A little time saver for those who are migrating their Symfony v4 projects to a newer versions

### CODE REVIEW

File: doc/upgrading/upgrade-v2.4-to-v3.0.md [hunk 1]
```diff
@@ -37,3 +37,22 @@ _Applies To: Projects with a custom `SoftDeleteableAdapter` implementation_
 
 The [`SoftDeleteableAdapter`](/src/SoftDeleteable/Mapping/Event/SoftDeleteableAdapter.php) interface has a new method
 for generating the DateTime value.
+
+## Translatable
+
+_Applies To: Projects migrated from Symfony v4 to v5_
```
Comment (phansys) [inline doc/upgrading/upgrade-v2.4-to-v3.0.md:43]: Where is the base documentation for this integration? What if there is no involved migration? Is it not required for users of Symfony v5?

Comment (much-rebel) [reply doc/upgrading/upgrade-v2.4-to-v3.0.md:43]: Base doc: https://github.com/doctrine-extensions/DoctrineExtensions/blob/main/doc/symfony4.md

Actually it is required for all of the Symfony v5 users, tho there is no base documentation for Syfmony 5.

**Comment by phansys** [comment]
Forcing new CI pipeline.

File: doc/upgrading/upgrade-v2.4-to-v3.0.md [hunk 2]
```diff
@@ -37,3 +37,22 @@ _Applies To: Projects with a custom `SoftDeleteableAdapter` implementation_
...
+
+In addition to the old config `is_bundle: false` needs to be added:
```
Comment (phansys) [inline doc/upgrading/upgrade-v2.4-to-v3.0.md:45]:
```suggestion
In addition to the existing config, `is_bundle: false` needs to be added:
```

**Comment by franmomu** [comment]
this looks like it is not related to migrate from 2.4 to 3, if it is only Symfony related should go in https://github.com/doctrine-extensions/DoctrineExtensions/blob/85f783195372b2ed03e410453528dd7452efe4f1/doc/symfony.md

**Pull request closed without merging**